[synthetics] Document how to import NPM packages and their limitations (#2277)
elastic/observability-docs: docs/en/observability/synthetics-command-reference.asciidoc

it using a _different_ ID. Select `yes` to create duplicates of all monitors in the project.
====

[NOTE]
====

> If the journey code in a project uses an external Node.js package,
> that package will be bundled with the journey code on `push`.
> However, you can't `push` bundles that are more than 800 Kilobytes.

Reviewer: ```suggestion
If the journey contains external NPM packages other than the `@elastic/synthetics`, those packages will be bundled along with the journey code when `push` command is invoked. However there are two limitations when using external packages

1. Bundled journey after compression should not be more than 800 Kilobytes.
2. Native node modules will not work as expected(platform inconsistency). 
```

Reviewer: Would replicate the same in the create test reference as well.

> ====

Author: I wasn't sure if I should also mention that "External packages that use native bindings will not work" in the command reference. Will it prevent the user from pushing the project or can they push it, but it won't work as expected?

it using a _different_ ID. Select `yes` to create duplicates of all monitors in the project.
====

[NOTE]
====
If the journey contains external NPM packages other than the `@elastic/synthetics`,
those packages will be bundled along with the journey code when the `push` command is invoked.
However there are some limitations when using external packages:
	
* Bundled journeys after compression should not be more than 800 Kilobytes.

> * Native node modules will not work as expected.
>   There will be platform inconsistency.

Reviewer: ```suggestion
* Native node modules will not work as expected due to platform inconsistency. 
```
